xml/article_sap_trento.xml

```diff
@@ -528,6 +528,41 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
     <para> The following subsections provides an overview of Trento's
       functionality. </para>
 
+    <section xml:id="sec-trento-web-home-page">
+      <title>Home page</title>
+      <para> The Home page allows you to determine at a glance the Health Status of your SAP environment. </para>
+      <figure>
+        <title>Hosts page</title>
+        <mediaobject>
+          <imageobject>
+            <imagedata width="80%" fileref="<home_page_screenshot>" />
```

Comment:
Here you should add the filename, otherwise you get an error. :slightly_smiling_face:

Reply:
It's now included.